Domain/TuitionPartnerEnquiry.cs

```diff
@@ -9,6 +9,10 @@ public class TuitionPartnerEnquiry
 
     public int MagicLinkId { get; set; }
 
+    public bool TuitionPartnerDecinedEnquiry { get; set; } = false;
+
+    public DateTime? TuitionPartnerDecinedEnquiryDate { get; set; }
```

Comment:
Typo here. It should be `TuitionPartnerDeclinedEnquiryDate `

Reply:
Good shout - have updated all instances of "Decined", including the migration script name, which will break this PR when tries to run it, so will create new PR to run the db migrations from scratch.